libnavigation-base/api/current.txt

```diff
@@ -854,9 +854,15 @@ package com.mapbox.navigation.base.speed.model {
     enum_constant public static final com.mapbox.navigation.base.speed.model.SpeedLimitSign VIENNA;
   }
 
-  public enum SpeedLimitUnit {
-    enum_constant public static final com.mapbox.navigation.base.speed.model.SpeedLimitUnit KILOMETRES_PER_HOUR;
-    enum_constant public static final com.mapbox.navigation.base.speed.model.SpeedLimitUnit MILES_PER_HOUR;
+  @Deprecated public enum SpeedLimitUnit {
+    enum_constant @Deprecated public static final com.mapbox.navigation.base.speed.model.SpeedLimitUnit KILOMETRES_PER_HOUR;
+    enum_constant @Deprecated public static final com.mapbox.navigation.base.speed.model.SpeedLimitUnit MILES_PER_HOUR;
+  }
+
+  public enum SpeedUnit {
```

Comment:
how was it hidden? 🤔 

cc @Guardiola31337 seems like an issue with metalava

Reply:
Someone forgot to update current.txt, it happens quite often lately.

Reply:
It may be due to `NAVAND-295`.